subprojects/docs/src/docs/userguide/gradle_wrapper.adoc

```diff
@@ -238,3 +238,41 @@ distributionSha256Sum=371cb9fbebbe9880d147f59bab36d61eee122854ef8c9ee1ecf12b8236
 ----
 
 Gradle will report a build failure in case the configured checksum does not match the checksum found on the server for hosting the distribution. Checksum Verification is only performed if the configured Wrapper distribution hasn't been downloaded yet.
+
+[[wrapper_checksum_verification]]
+== Verifying the checksum of the Gradle Wrapper JAR
```

Comment:
I would start this section by explaining why a reader might want to do this. How is it different from verifying the Gradle distribution downloaded by the wrapper?